src/smart-components/request/request-detail/action-transcript.js

```diff
@@ -6,28 +6,27 @@ import {
   Text,
   TextVariants
 } from '@patternfly/react-core';
-import { CheckCircleIcon, CommentIcon, EnvelopeIcon, OutlinedTimesCircleIcon, AngleDoubleRightIcon } from '@patternfly/react-icons';
-
-const operationDisplayName = { memo: 'Comment from', approve: 'Approved By',
-  deny: 'Denied by', notify: 'Notified By', skip: 'Skipped by', cancel: 'Canceled by' };
+import { CheckCircleIcon, CommentIcon, EnvelopeIcon, OutlinedTimesCircleIcon, AngleDoubleRightIcon, OnRunningIcon } from '@patternfly/react-icons';
+const operationDisplayName = { memo: 'Comment from', approve: 'Approved by',
+  deny: 'Denied by', notify: 'Notified by', skip: 'Skipped by', cancel: 'Canceled by', start: 'Started by' };
 
 const operationIcon = {
   memo: <CommentIcon/>,
   approve: <CheckCircleIcon style={ { color: 'var(--pf-global--success-color--100)' } }/>,
   deny: <OutlinedTimesCircleIcon style={ { color: 'var(--pf-global--danger-color--100)' }	}/>,
   notify: <EnvelopeIcon/>,
-  skip: <AngleDoubleRightIcon/>
+  skip: <AngleDoubleRightIcon/>,
+  start: <OnRunningIcon/>
 };
 
 export const ActionTranscript = actionList => {
-  // TODO - remove actionList.actionList after the new Apis are working - it is only for old style requests
   const actions = actionList.data || actionList.actionList.data;
   return (
     <Stack>
       { actions.map(actionItem =>
         <div key={ `${actionItem.id}-action` }>
           <TextContent><Text style={ { marginBottom: 0 } } className="data-table-detail content" component={ TextVariants.small }>
-            { `${timeAgo(actionItem.created_at)}` }
+            { timeAgo(actionItem.created_at) }
```

Comment:
Can we use the DateFormat component here too?

Reply:
@Hyperkid123 - the timeAgo in helpers.js is using the DateFormat.

Reply:
AH sorry